portfolio/src/main/java/com/google/sps/data/Task.java

```diff
@@ -53,7 +52,22 @@ public Duration getDuration() {
     return duration;
   }
 
-  public Integer getPriority() {
-    return priority.getPriority();
+  public TaskPriority getPriority() {
+    return priority;
+  }
+
+  // These methods provide a way to tell if two objects are both Task objects
+  // and have all the same fields
+
+  @Override
+  public boolean equals(Object other) {
+    return other instanceof Task && equals(this, (Task) other);
+  }
+
+  private static boolean equals(Task a, Task b) {
```

Comment:
I think you'll want to start using task ID soon. The current approach won't work well when you begin to integrate with Tasks API.